source/extensions/network/dns_resolver/getaddrinfo/getaddrinfo.cc

```diff
@@ -194,8 +194,7 @@ void GetAddrInfoDnsResolver::resolveThreadRoutine() {
                   *num_retries);
         next_query->addTrace(static_cast<uint8_t>(GetAddrInfoTrace::DoneRetrying));
         response = std::make_pair(ResolutionStatus::Failure, std::list<DnsResponse>());
-      } else if (treat_nodata_noname_as_success &&
-                 (rc.return_value_ == EAI_NONAME || rc.return_value_ == EAI_NODATA)) {
+      } else if (rc.return_value_ == EAI_NONAME || rc.return_value_ == EAI_NODATA) {
```

Comment:
looks like a functional change as well as the flag flip can you update PR description?
Also there's some formal metadata for "this fixes an issue introduced in PR/hash X" which should also be included.  if you can't dig it up someone on EPT will likely know what it is
cc @yanavlasov for import awareness
/wait-any

Reply:
I updated the PR description.